plugin-server/src/types.ts

```diff
@@ -87,19 +87,12 @@ export enum PluginServerMode {
     recordings_blob_ingestion = 'recordings-blob-ingestion',
 }
 
-export const stringToPluginServerMode: { [key: string]: PluginServerMode } = {
-    ingestion: PluginServerMode.ingestion,
-    'ingestion-overflow': PluginServerMode.ingestion_overflow,
-    async: PluginServerMode.plugins_async,
```

Comment:
I think this doesn't work for `async` which is mapped to `plugins_async` because didn't want to use keywords in the type

Reply:
Have added a test for this.